File: LeetCode/Easy/0771-jewels-and-stones/0771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        vector<bool> types(128);
        int ret = 0;
        
        for (char jewel : jewels) types[jewel] = true;
        
        for (char stone : stones)
            if (types[stone]) ++ret;
        
        return ret;
    }
};

File: LeetCode/Easy/0771-jewels-and-stones/0771-jewels-and-stones.go
func numJewelsInStones(jewels string, stones string) int {
    types := make([]bool, 128)
    ret := 0
    
    for _, jewel := range jewels { types[jewel] = true }
    
    for _, stone := range stones {
        if types[stone] { ret++ }
    }
    
    return ret
}